lib/in_memory_data_store.rb

```diff
@@ -85,6 +88,14 @@ def get_latest_entry_number(entry_type)
       entry.nil? ? 0 : entry.entry_number
     end
 
+    def get_root_hash
+      get_latest_entry_number(:user) > 0 ? @data[:root_hash] : EMPTY_ROOT_HASH
```

Comment:
What does it mean if I get a `root_hash` that is an empty string? Based on the initialisation of `@data` that seems possible.

Reply:
It shouldn't be possible. Let me address that.

Reply:
See new commits @arnau